django/views/static.py

```diff
@@ -111,7 +110,7 @@ def directory_index(path, fullpath):
     return HttpResponse(t.render(c))
 
 
-def was_modified_since(header=None, mtime=0, size=0):
+def was_modified_since(header=None, mtime=0):
```

Comment:
At least https://github.com/jazzband/django-downloadview/blob/b64b1ad21a8ceeb7da05864ec57432853fd6b79b/django_downloadview/views/base.py#L124 seems to be using it. That one is easily fixed (and other probably as well) but begs the question whether we should deprecate it…

Reply:
I added release notes.